chemreg/openapi/schemas/substance/synonymQuality.libsonnet

```
@@ -0,0 +1,33 @@
+ {
+  app: 'Substance',
+  type: 'synonymQuality',
+  typePlural: 'synonymQualities',
+  description: 'Everything about Synonym Qualities.',
+  attributes: {
+    name: {
+      type: 'string',
+      maxLength: 49,
+      pattern: '^[a-z0-9-]+$',
+    },
+    label: {
+      type: 'string',
+      maxLength: 99,
+    },
+    shortDescription: {
+      type: 'string',
+      maxLength: 499,
+    },
+    longDescription: {
+      type: 'string',
+    },
+    scoreModifier: {
```

```suggestion
    scoreWeight: {
```
This line is named incorrectly.